src/types/Datum.ts

```diff
@@ -0,0 +1,5 @@
+export interface Datum {
```

Comment:
It's used by the DatumRegistry contract, not yet being used by the code present in this repository, but being part of the project I thought it would be better to have this than to hide it. I'm wondering if anyone would need to use this interface in their contracts. They can, but not sure if they would.

Reply:
I don't understand why we need this if we're not exporting in `index.ts`. I think if the only reason we're keeping it is "we might need it in the future", we should remove and leave it to the future.

Reply:
You're right. This specific interface is not used. I was talking about the solidity interfaces, not this one. I'll delete this one.